x/feegrant/types/grant_test.go

```diff
@@ -49,6 +51,12 @@ func TestGrant(t *testing.T) {
 	})
 	require.NoError(t, err)
 
+	zeroAllowance, err := types.NewFeeAllowanceGrant(addr, addr2, &types.BasicFeeAllowance{
```

Comment:
maybe all these `NewFeeAllowanceGrant`s test could be turned into table tests (using the one below)?

Reply:
yeah that will be cleaner, [uses table tests now](https://github.com/cosmos/cosmos-sdk/pull/9193/files#diff-81a8c7816ea8afb0c36560c24ff72cff1940b107486ab8b59eb3fb68eb4275c2R24-R28)